src/main/java/org/octri/hpoonfhir/config/FhirConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.octri.hpoonfhir.config;
+
+import org.octri.hpoonfhir.service.FhirService;
+import org.octri.hpoonfhir.service.Stu2FhirService;
+import org.octri.hpoonfhir.service.Stu3FhirService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class FhirConfig {
+	
+	@Bean
+	public FhirService r3FhirService() {
+		return new Stu3FhirService("https://r3.smarthealthit.org");
+	}
+
+	@Bean
+	public FhirService epicFhirService() {
+		return new Stu2FhirService("https://open-ic.epic.com/FHIR/api/FHIR/DSTU2");
+	}
```

Comment:
It might make sense to make the urls properties that can be set in the config.

Reply:
👍

Reply:
Yes, these will move to config at some point. I didn't see any need for it yet with only one parameter. At some point we'll introduce OAuth, and this will get more complex.